test/compile/common.test.ts

```diff
@@ -3,94 +3,104 @@
 import {assert} from 'chai';
 import {parseUnitModel} from '../util';
 import {X} from '../../src/channel';
-import {timeFormat, formatMixins, applyColorAndOpacity} from '../../src/compile/common';
+import {formatMixins, applyColorAndOpacity} from '../../src/compile/common';
+import {format} from '../../src/timeunit';
 
 describe('Model', function() {
-  describe('timeFormat()', function() {
+  describe('format()', function() {
     it('should get the right time template', function() {
```

Comment:
I would break this `it` into multiple `it` since this is testing many aspects of the method.  
(So that you can use `const` with the `model` below instead of `let`.